modules/Hoot/tools/selectBbox.js

```diff
@@ -200,6 +200,34 @@ export default class SelectBbox extends EventEmitter {
                     bboxOptions.selectAll( 'button' )
                         .classed( 'selected', false );
                 });
+
+            // construct input section for user custom overpass queries
+            this.userInputContainer = this.form
+                .select( '.wrapper div' )
+                .insert( 'div', '.modal-footer' )
+                .classed( 'button-wrap user-input', true );
+
+            this.userInputContainer.append( 'input' )
+                .attr( 'type', 'checkbox' )
+                .on('click', function() {
+                    const isChecked = d3.select( this ).property( 'checked' );
+                    customQueryInput.classed( 'hidden', !isChecked );
+                });
+
+            this.userInputContainer.append('div').text('Use custom query for public overpass data');
+
+            const placeholder = 'Example Query:\n' +
+                '[out:json][bbox:{{bbox}}];\n' +
+                '(\n' +
+                '   node;<;>;\n' +
+                ')->.all;\n' +
+                '.all;\n' +
+                'out meta;\n' +
+                '>;\n';
```

Comment:
This is what Overpass Turbo uses for simple bbox call
```
/*
This is a simple map call.
It returns all data in the bounding box.
*/
[out:xml];
(
  node({{bbox}});
  <;
);
out meta;
```

what's the diff?

Reply:
it doesnt recurse back down so you can potentially lose out on relations that may not have nodes.

the bbox declaration at the top is nice because you dont need to keep stating `({{bbox}})`

I will clean the example up a little though